src/app/learn/reference/cli-commands/install/index.html

```diff
@@ -0,0 +1,8 @@
+<stache
+  pageTitle="install">
+
+  <p>
+    <stache-code>skyux install</stache-code> deletes the <stache-code>node_modules</stache-code> folder and runs <stache-code>npm install</stache-code>. This clears the existing dependencies in the <stache-code>node_modules</stache-code> folder and ensures that all SKY UX dependencies are consistent and up-to-date.
```

Comment:
I'm worried the "up-to-date" may cause ambiguity.  Perhaps we could add "up-to-date based on your `package.json`" or remove it all together.  Just don't want anyone to think that this command updates the versions, it simply installs the versions that match whatever's specified in their `package.json`.

Reply:
Not sure when you are going to get another chance to look at this, so I'm going to go with the latter option.